### PR TITLE
drop pegasus.cdo_educate table

### PR DESCRIPTION
**Pull request by tanyaparker**
- [x] delete actual gsheet from pegasus drive after merging

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/educate/index.haml [hunk 1]
```diff
@@ -8,13 +8,24 @@ title: Educator Overview
   %img.educate-banner{src: "/images/homepage/fill-945x300/girl10.jpg"}
 
 #educate-curriculum-grid
-  - DB[:cdo_educate].where(section_s:'curriculum').each do |item|
-    = view :educate_box, item: item, color: "#59b9dc"
+  - curriculum_boxes = []
```
Comment (ashercodeorg) [inline pegasus/sites.v3/code.org/public/educate/index.haml:11]: Can (should) this ruby be moved to a ruby helper file?

File: pegasus/sites.v3/code.org/views/educate_box.haml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 -color ||= "#59b9dc"
+-item[:button_s] ||= 'Learn more'
```
Comment (ashercodeorg) [inline pegasus/sites.v3/code.org/views/educate_box.haml:2]: Is this change intentional? Why is it part of this PR?

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/views/educate_box.haml:2]: Since they were all the same I thought it'd be unnecessarily repetitive to put 'Learn more' explicitly for each box. If we were to be purist about it I would have put all the table cleanup/refactoring in a previous PR... I felt like this was related enough to be together.

File: pegasus/sites.v3/code.org/public/educate/index.haml (hunk 1 above)
Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/educate/index.haml:11]: Is your philosophy that if we do the same task at least twice it could be a helper? So you think this merits its own helper?

Comment (ashercodeorg) [reply pegasus/sites.v3/code.org/public/educate/index.haml:11]: Primarily, I'm trying to decrease the amount of ruby (JS, CSS) in HTML and HAML files.

File: pegasus/sites.v3/code.org/views/educate_box.haml (hunk 2 above)
Comment (ashercodeorg) [reply pegasus/sites.v3/code.org/views/educate_box.haml:2]: Roger, I just didn't (still don't) see where the `Learn more` button was previously.

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/views/educate_box.haml:2]: `button_s` was one of the columns of the gsheet so each box could have a different button label. But they were all `Learn more` so instead of transferring that data into the curriculum_boxes and resources_boxes rows I made it the default.

File: pegasus/sites.v3/code.org/public/educate/index.haml (hunk 1 above)
Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/educate/index.haml:11]: Oh, I thought we just wanted to separate JS/CSS into different files. Why do we want to limit ruby?

Comment (ashercodeorg) [reply pegasus/sites.v3/code.org/public/educate/index.haml:11]: For the same reasons as JS / CSS. Better design, allows separation of concerns, allows linting of ruby, etc.
